pkg/sql/parser/testdata/create_misc

```diff
@@ -95,8 +94,7 @@ CREATE STATISTICS a ON col1 FROM t WITH OPTIONS THROTTLING 0.1 AS OF SYSTEM TIME
 ----
 CREATE STATISTICS a ON col1 FROM t WITH OPTIONS THROTTLING 0.1 AS OF SYSTEM TIME '2016-01-01'
 CREATE STATISTICS a ON col1 FROM t WITH OPTIONS THROTTLING 0.1 AS OF SYSTEM TIME ('2016-01-01') -- fully parenthesized
-CREATE STATISTICS a ON col1 FROM t WITH OPTIONS THROTTLING _ AS OF SYSTEM TIME _ -- literals removed
-REPARSE WITHOUT LITERALS FAILS: at or near "_": syntax error
+CREATE STATISTICS a ON col1 FROM t WITH OPTIONS THROTTLING 0.001 AS OF SYSTEM TIME '_' -- literals removed
```

Comment:
is this the output you get?

when i do the rewrite i get
```
CREATE STATISTICS a ON col1 FROM t WITH OPTIONS THROTTLING 0.001 AS OF SYSTEM TIME _ -- literals removed
```

Reply:
I get the same, I must have messed up the rebase.